overlay.d/07el9/etc/ssh/sshd_config.d/41-rhcos-defaults.conf

```diff
@@ -0,0 +1,6 @@
+# Disable PasswordAuthentication in SSH
```

Comment:
Let's change this comment to now explicitly link to https://issues.redhat.com/browse/OCPBUGS-11613

And also call out that this is a change we need to carry over because we did this in the RHEL8 timeframe, but we could potentially drop it by default in the future (for example, by only disabling password logins for the `core` user).